plot_explorer.py

```diff
@@ -213,7 +231,7 @@ def createMenuBar(self):
         self.maskingAction.setToolTip('Toggle masking')
         self.maskingAction.setStatusTip('Toggle whether masking is enabled')
         self.maskingAction.triggered[bool].connect(lambda bool=bool:
-            self.toggleMasking(bool, apply=True))
+                                                   self.toggleMasking(bool, apply=True))
```

Comment:
So I haven't really dug into the code much, but there's some serious confusion with the use of `bool` here:
- The `bool` class is being used to index the `.triggered` method. The documentation for it says that it should receive a boolean _value_. Passing the `bool` class will be interpreted as `True` (since a class object is considered a non-empty object).
- The lambda takes a default argument with the `bool` class, which again is interpreted as `True`. You'll notice in the `toggleMasking` method it's necessary to explicitly convert the value passed to a bool. If the default argument here were `bool=True`, that would avoid the need for `bool(...)` in `toggleMasking`. Also, the name of the lambda argument should not be `bool` (maybe `checked` instead since that's what the signal is sending).

I'm pretty sure that if you have `triggered[True]`, this means that the checked state will always be passed to the slot. Thus, the lambda argument shouldn't need a default. The following change should have the same effect as the current code:
```Python
self.maskingAction.triggered[True].connect(lambda checked: self.toggleMasking(checked, apply=True))
```
I'd also consider using `functools.partial` for some of the slots instead of lambda:
```
from functools import partial
self.maskingAction.triggered[True].connect(partial(self.toggleMasking, apply=True))

Reply:
Thanks for digging into this! This is old code. I was just updating indentation here, but this should definitely be addressed. I'll take care of it here. 

Agreed about using `partial`. I use that elsewhere and can apply it here as well.

Reply:
Looks like we should be using `*Action.toggled` to connect to an event in these cases. Updated that as well.